aws-datastore/src/test/resources/conflict-unhandled-response.json

```diff
@@ -0,0 +1,29 @@
+{
+  "data": {
+    "updateNote": null
+  },
+  "errors": [
+    {
+      "path": [
+        "updateNote"
+      ],
+      "data": {
+        "id": "KoolId22",
+        "content": "Resurecting the dataz",
+        "_version": 7,
+        "_deleted": true,
+        "_lastChangedAt": 1601499066604
```

Comment:
`Temporal.Timestamp` should always be in seconds, not milliseconds.

Reply:
I believe this value was observed in the wild, in a real response from AppSync. Will confirm.

Reply:
We discussed this offline. Our findings:

1. [AppSync returns this field in milliseconds](https://docs.aws.amazon.com/appsync/latest/devguide/conflict-detection-and-sync.html), in the conflict handler.
2. But, in a generated schema, `_lastChangedAt`  is represented as an `AWSTimestamp`.  And an `AWSTimestamp` [is defined as number of seconds since 1970](https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html).